.travis/run.sh

```diff
@@ -54,6 +54,13 @@ else
             pip install -e .
             pytest -m local -l
             ;;
+        dynamodb-encryption-sdk)
+            git clone --depth=1 https://github.com/awslabs/aws-dynamodb-encryption-python
+            cd aws-dynamodb-encryption-python
+            pip install -r test/requirements.txt
+            pip install -e .
+            pytest -m "local and not slow and not veryslow and not nope" -l
```

Comment:
I lol'd

Reply:
Haha, yup. Of the many levels of hypothesis-powered combinatorically exploding test scenarios, I figure we want the smallest set here.